[core]/es_extended/server/main.lua

```diff
@@ -122,252 +122,211 @@ if not Config.Multichar then
 	end)
 end
 
-function loadESXPlayer(identifier, playerId, isNew)
+function loadESXPlayer(identifier, playerId, isNew) 
 	local userData = {
-		accounts = {},
-		inventory = {},
-		job = {},
-		loadout = {},
-		playerName = GetPlayerName(playerId),
-		weight = 0,
-		metadata = {}
-	}
-	local result = MySQL.prepare.await(loadPlayer, { identifier })
-	local job, grade, jobObject, gradeObject = result.job, tostring(result.job_grade)
-	local foundAccounts, foundItems = {}, {}
-
-	-- Accounts
-	if result.accounts and result.accounts ~= '' then
-		local accounts = json.decode(result.accounts)
-
-		for account, money in pairs(accounts) do
-			foundAccounts[account] = money
-		end
-	end
-
-	for account, data in pairs(Config.Accounts) do
-		if data.round == nil then
-			data.round = true
-		end
-		local index = #userData.accounts + 1
-		userData.accounts[index] = {
-			name = account,
-			money = foundAccounts[account] or Config.StartingAccountMoney[account] or 0,
-			label = data.label,
-			round = data.round,
-			index = index
-		}
-	end
-
-	-- Job
-	if ESX.DoesJobExist(job, grade) then
-		jobObject, gradeObject = ESX.Jobs[job], ESX.Jobs[job].grades[grade]
-	else
+        accounts = {}, 
+        inventory = {}, 
+        loadout = {}, 
+        playerName = GetPlayerName(playerId), 
+        weight = 0, 
+		metadata = {}, 
+
+		firstName = 'John', 
+		lastName = 'Doe', 
+		dateofbirth = '01/01/2000', 
+		height = 120, 
+		skin = { sex = 'm' }, 
+        sex = 'm', 
```

Comment:
What is this? Duplicate?

Reply:
the skin is for skinchanger and / or esx skin 
and sex is for identity its to maintain backwards compatibility 
what we could do is always fallback to the skin sex